test/matchConfigTest.cpp

```diff
@@ -8,34 +8,36 @@
 #include <gtest/gtest.h>
 #include <matchconfig/MatchConfig.hpp>
 
-TEST(MatchConfigDecoding, MatchConfig) {
+TEST(MatchConfigDecodingEncoding, MatchConfig) {
```

Comment:
```suggestion
TEST(MatchConfig, MatchConfigDecodingEncoding) {
```
`TEST(TestSuiteName, TestName)`

[here](https://github.com/google/googletest/blob/master/googletest/docs/primer.md#simple-tests)